sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -1250,6 +1313,9 @@ def test_sdf_with_sdf_initiated_checkpointing(self):
   def test_sdf_with_watermark_tracking(self):
     raise unittest.SkipTest("This test is for a single worker only.")
 
+  def test_deduplication_transform_with_processing_time(self):
+    raise unittest.SkipTest("This test is for a single worker only.")
```

Comment:
Is test stream unsupported in these other setups?

Reply:
No, I don't think so.